Day-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,29 @@
+.btn1 {
+	width: 113.25px;
+	height: 92px;
+	border-width: 1px;
+}
+
+.btn2 {
+	width: 339.75px;
+	height: 92px;
+	border-width: 1px;
+}
+
+.white-btn {
+	background: white;
```

Comment:
Try not to mix names and #numbers for colors try to chose one and stick to it as in #fff for white